From 4ad69d9938ab3437a247dcf5ddcf71c7fdf02325 Mon Sep 17 00:00:00 2001
From: Heinrich Schuchardt <xypron.glpk@gmx.de>
Date: Sun, 14 Oct 2018 20:35:53 +0200
Subject: [PATCH 1/1] sandbox: remove stray DEBUG

DEBUG should not be defined in production code.
Change printf() to debug() where this writes a debug message.

Signed-off-by: Heinrich Schuchardt <xypron.glpk@gmx.de>
---
 arch/sandbox/cpu/cpu.c | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/arch/sandbox/cpu/cpu.c b/arch/sandbox/cpu/cpu.c
index 6098945049..fdfb209f77 100644
--- a/arch/sandbox/cpu/cpu.c
+++ b/arch/sandbox/cpu/cpu.c
@@ -2,7 +2,7 @@
 /*
  * Copyright (c) 2011 The Chromium OS Authors.
  */
-#define DEBUG
+
 #include <common.h>
 #include <dm.h>
 #include <errno.h>
@@ -105,8 +105,8 @@ void *phys_to_virt(phys_addr_t paddr)
 	state = state_get_current();
 	list_for_each_entry(mentry, &state->mapmem_head, sibling_node) {
 		if (mentry->tag == paddr) {
-			printf("%s: Used map from %lx to %p\n", __func__,
-			       (ulong)paddr, mentry->ptr);
+			debug("%s: Used map from %lx to %p\n", __func__,
+			      (ulong)paddr, mentry->ptr);
 			return mentry->ptr;
 		}
 	}
@@ -152,7 +152,7 @@ phys_addr_t virt_to_phys(void *ptr)
 		       __func__, ptr, (ulong)gd->ram_size);
 		os_abort();
 	}
-	printf("%s: Used map from %p to %lx\n", __func__, ptr, mentry->tag);
+	debug("%s: Used map from %p to %lx\n", __func__, ptr, mentry->tag);
 
 	return mentry->tag;
 }
-- 
2.19.1

